python/baseline/services.py

```diff
@@ -454,18 +386,17 @@ def predict(self, tokens, **kwargs):
         mxwlen = kwargs.get('mxwlen', 40)
 
         for k, vectorizer in self.vectorizers.items():
```

Comment:
Why is the `vectorizer.mxlen` still getting manually set here? Shouldn't this be handled by the `prepare_vectorizers` method?

Reply:
made the change that `mxlen` is set by `prepare_vectorizers`